.github/workflows/check.yml

```diff
@@ -127,3 +127,51 @@ jobs:
           file: lcov.info
           fail_ci_if_error: false
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  qns-docker-image: # Docker image for QUIC network simulator i.e. interop runner (https://github.com/quic-interop/quic-interop-runner)
+    needs: check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
```

Comment:
```suggestion
```

https://github.com/docker/build-push-action#git-context

Reply:
Cool. This was news to me.

e2af258cd7b5670357e023ce759176d55d9d68c8